cib/remote.c

```diff
@@ -324,7 +324,9 @@ cib_remote_listen(gpointer data)
 
     crm_client_init();
     new_client = calloc(1, sizeof(crm_client_t));
+    CRM_ASSERT(new_client != NULL);
     new_client->remote = calloc(1, sizeof(crm_remote_t));
```

Comment:
I think a log message is more helpful to a sysadmin than a core dump if a problem does occur, though I agree that the chances of the issue are tiny.